src/components/header/__test__/Header.test.js

```diff
@@ -0,0 +1,14 @@
+import { render, screen } from '@testing-library/react';
+import Header from '../Header';
+
+describe('header testing', () => {
+  it('shoud be in the document', () => {
+    render(<Header />);
+    expect(screen.getByTestId('header')).toBeInTheDocument();
+  });
+
+  it('shoud be visible in the document', () => {
+    render(<Header />);
+    expect(screen.getByTestId('header')).toBeVisible();
+  });
+});
```

Comment:
**Testing**
![image](https://github.com/AjroudiRida/fruittree/assets/78251927/90586af2-5dde-4712-9a4d-3637e3e52162)
![image](https://github.com/AjroudiRida/fruittree/assets/78251927/c506108a-fa42-497c-891a-89d2680199a2)
![image](https://github.com/AjroudiRida/fruittree/assets/78251927/12744f68-bf38-49ac-9a88-75d6cb61b0c8)

- [ ] Please ensure that all the tests are passing, The main objective of unit testing is to isolate written code to test and determine if it works as intended which is an important step in the development process. 🛠😊

- [ ] If done correctly, unit tests can detect early flaws in code which may be more difficult to find in later testing stages.